src/main/java/com/redhat/red/build/koji/model/xmlrpc/KojiImageBuildInfo.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc. (jcasey@redhat.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.red.build.koji.model.xmlrpc;
+
+import org.commonjava.rwx.binding.anno.DataKey;
+import org.commonjava.rwx.binding.anno.StructPart;
+
+@StructPart
+public class KojiImageBuildInfo
```

Comment:
I've been thinking about this more, and I agree. I guess I'm a little hesitant to make alternatives to KojiBuildInfo, though, particularly when all of these builds should still have (to some extent or other) the same fields. Many fields are optional or can have a wide variety of information in them, it seems. I'd just prefer to try to use KojiBuildInfo as the basis for them all, unless there's some compelling subset used by a particular type, where the rest of the fields would just be confusing.